Update various roles for SR2019 (#26)
srobo/runbook: docs/competition/volunteers/refreshments.md

# Refreshments

It's important that volunteers stay hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).

> *In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left.*

Reviewer: water points are on bar 2 and bar 3

Author: Fixed in 49d401e

> **In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left. Also Water points are on both level 2 and 3 in the corner furthest from the toilets.**

Reviewer 2: ```suggestion
**In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
```

Author: I've been learning too much German it seems

It is the responsibility of the Refreshments role to offer and provide refreshments to all volunteers, especially those who can't leave their roles.

To do this, you should grab a tray, fill it with a mix of drinks and food, and go to all volunteer positions, offering refreshments.
Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.

> Please don't forget about those at tech decks, those at reception/helpdesk, and those who are commentating/marshalling.

Reviewer 2: Could we be explicit about why this is the case?
```suggestion
People at the following locations will be unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
- tech desks
- reception
- helpdesk
- arena: commentator, marshals, head shepherd, score entry

The tech desks roles and commentator are particularly important as they do not rotate between people at all.
```

Author: Thanks for the fix!

## Running low on refreshments

> If we're running low on refreshments, it's your responsiblity to either go or find someone to go get more from the nearby shops.

Reviewer: budget must be approved first

Author: Fixed in 49d401e

Reviewer 2: Might a better approach be to pre-allocate a small amount of petty-cash for this, maybe even have that as cash stored somewhere to hand?

I agree we'd want to have someone check this, but I think we can mostly trust our volunteers and if we do run out we'll want to be able to replace it fairly promptly. If the treasurer is busy (quite likely given that we currently have a lot of role overlap) then getting explicit approval here could easily take a while.

Author: Yeah I'm happy to lay down £50 next to the refreshments and trust that people won't steal anything

Author: Fixed in eeb14dd1d7bdf94419f50f4ff571d01f970572cb and 3f0530c941d3a115a8c4c08c53a274b0533d24ae

Reviewer: I would suggest we evaluate the refreshments levels at periodic points and decided to order more as we will have a good idea of how fast we are getting through them long before they run out.

The nearest food shopfor SR2019 is the 'Coop Food' on Burgess road, see: https://goo.gl/maps/VETqx4apQet

We recommend the items you get are:
- Coffee (if we have cups)
- Plastic tumblers
- Soft drinks
- Fruit
- Biscuits

> Keep hold of your receipts, and send them in when we ask for Travel reimbursements, repayments will be made at that time.

Reviewer 2: ```suggestion
Keep hold of your receipts, and send them in when we ask for travel reimbursements, repayments will be made at that time.
```

# Refreshments

It's important that volunteers stay hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).

> **In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**

Reviewer 2: ```suggestion
**In SR2019, the refreshments are in the Volunteer Rest room, which is on level 2 past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
```

It is the responsibility of the Refreshments role to offer and provide refreshments to all volunteers, especially those who can't leave their roles.

To do this, you should grab a tray, fill it with a mix of drinks and food, and go to all volunteer positions, offering refreshments.
Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.

People at the following locations will be unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
- tech desks
- reception
- helpdesk
- arena: commentator, marshals, head shepherd, score entry

The tech desks roles and commentator are particularly important as they do not rotate between people at all.

## Running low on refreshments

> If we're running low on refreshments, it's your responsiblity to either go or find someone to go get more from the nearby shops. There should be an amount of cash by the refreshments to spend to replenish. It's intended to last for the whole competition, so do not spend it all. If the money has ran out, please speak to the Volunteer Manager.

Reviewer: given how far coop is away from the venue we should try and avoid this trip being made as it is quite a long way away and would take the majority of the slot. A possible solution would be to have them check stock at the beginning of a shift and report if it is running low we can then send a free volunteer to purchase it.

Author: Fixed in 547ece8f01e7bbf3865f009d31a49035c29c6c4d

> The nearest food shopfor SR2019 is the 'Coop Food' on Burgess road, see: https://goo.gl/maps/VETqx4apQet

Reviewer 3: ```suggestion
The nearest food shop for SR2019 is the 'Coop Food' on Burgess road, see: https://goo.gl/maps/VETqx4apQet
```

> We recommend the items you get are:

Reviewer 3: ```suggestion
Items we should keep in constant supply:
```
